src/nodes/cls_node.c

```diff
@@ -130,6 +167,11 @@ static __rte_always_inline rte_edge_t get_next_index(__rte_unused struct rte_nod
 	struct dp_virtsvc *virtsvc;
 #endif
 
+#ifdef ENABLE_PF1_PROXY
+	if (pf1_tap_proxy_forward(m))
```

Comment:
What about an unlikely here ?

Reply:
Actually if anything shouldn't this be `likely`? If you are compiling with proxy, the only reason to do so is to run it (or test it but in that case optimization does not matter).

Reply:
Well, this is classify node right ? When we enable proxy and we have 100 VMs running traffic on top of dpservice with their workloads and there is "one" PF1 interface where we check whether the traffic needs to be forwarded to TAP proxy port. At any given time in classify node, is it more likely to be a PF1 traffic that we classify or is it the traffic from one of the VF ports ?